hack/olm-registry/olm-artifacts-template.yaml

```
@@ -149,3 +152,14 @@ objects:
         name: ${REPO_NAME}
         source: ${REPO_NAME}-registry
         sourceNamespace: openshift-${REPO_NAME}
+    - apiVersion: v1
+      data:
+        avo_config.yaml: |-
+          apiVersion: avo.openshift.io/v1alpha1
+          kind: AvoConfig
+          enableVpcEndpointController: true
+          enableVpcEndpointAcceptanceController: ${ENABLE_VPC_ACCEPTANCE}
+      kind: ConfigMap
+      metadata:
+        name: avo-config
+        namespace: openshift-aws-vpce-operator
```

Probably no difference, but for consistency...
```suggestion
        namespace: openshift-${REPO_NAME}
```

---

Updated all instances to match now